pkgs/test/default.nix

```diff
@@ -37,6 +37,8 @@ with pkgs;
 
   cross = callPackage ./cross {};
 
+  php = recurseIntoAttrs (callPackages ./php {});
```

Comment:
What's the benefit of having the tests in `pkgs/test/default.nix` instead of in `nixos/tests/php/`? In the latter one we expose it in the attribute `php.tests` - which also would make the tests run for each PHP version.

This attribute is also usually built by ofborg.

Reply:
Less expensive tests. There's IMHO no need for a QEMU VM for such a "small" testcase.

Reply:
Sure, makes sense, also there's no real need to test for different versions here I would think either since this would mostly cover the PHP code itself.